Doc/whatsnew/3.11.rst

```diff
@@ -365,6 +365,11 @@ Other Language Changes
 * Add :option:`-P` command line option and :envvar:`PYTHONSAFEPATH` environment
   variable to not prepend a potentially unsafe path to :data:`sys.path` such as
   the current directory, the script's directory or an empty string.
+
+  Add also the :option:`-p` option to prepend the potentially unsafe path to
+  :data:`sys.path`: ignore the :option:`-P` option and the
+  :envvar:`PYTHONSAFEPATH` environment variable. The :option:`-p` option takes
+  precedence over the :option:`-I` option (isolated mode).
```

Comment:
As before, I'd suggest being clearer here that `-p` doesn't override `-I` entirely, just the `-P` it implies.

```suggestion
  :envvar:`PYTHONSAFEPATH` environment variable. The :option:`-p` option also
  takes precedence over the :option:`-P` implied by the :option:`-I` option
  (isolated mode).
```